app/views/households/_members_table_row.html.haml

```diff
@@ -12,5 +12,5 @@
         = icon(:'trash-alt')
 
   %td
-    %strong= link_to(member.person, member.person)
+    %strong= link_to(member.person, person_home_path(member.person), data: {turbo_frame: :_top})
```

Comment:
Eventuell könnte man den Link (target und data attribute) noch in der view spec ebenfalls testen (analog https://github.com/hitobito/hitobito/blob/1eb3ca660ce6a91ecf0c5e17ad7d771d34567ac3/spec/views/households/edit.html.haml_spec.rb#L76) aber die feature spec prüft das implizit auch, also nice2have aber nicht zwingend notwendig

Reply:
Finde das ein wenig tautologisch wenn man in der spec überprüft ob ein data attribute da ist. Die spec soll ja überprüfen ob der Link funktioniert und bei Turbo issues ist das nur in einem feature spec möglich, da Javascript.